app/controllers/hiring_staff/vacancies/application_details_controller.rb

```diff
@@ -49,9 +49,10 @@ def update
   private
 
   def application_details_form_params
-    params.require(:application_details_form).permit(:application_link, :contact_email,
+    params.require(:application_details_form).permit(:application_link, :contact_email, :expiry_time,
                                                      :expires_on_dd, :expires_on_mm, :expires_on_yyyy,
-                                                     :publish_on_dd, :publish_on_mm, :publish_on_yyyy)
+                                                     :publish_on_dd, :publish_on_mm, :publish_on_yyyy,
+                                                     :expiry_time_hh, :expiry_time_mm, :expiry_time_meridian)
```

Comment:
We're using a mixture of `meridian` and `meridiem`.  Can we standardise on the latter?

Reply:
I agree that we should standardise on [meridiem](https://en.wikipedia.org/wiki/12-hour_clock). 👍

Reply:
Resolved this in e543ec7